### DISCUSSION TITLE
WSL supported?

### DISCUSSION THREAD

**Discussion by Ismoh** [nektos/act#1255] [Q&A] [answered]
I guess it's possible to run act on windows wsl, but still thought to ask.

I am going to try it out and let you know!

**Comment by Ismoh** [comment, marked as answer]
Looks like it is working:
I did this commands on wsl1 windows 10:
```bash
mkdir github-actions-locally
cd github-actions-locally/
curl https://raw.githubusercontent.com/nektos/act/master/install.sh | sudo bash
export PATH=$PATH:/mnt/d/github-actions-locally/bin
cd /your_path_to_your_local_repo_directory/
act -l
```

I also put `export PATH=$PATH:/mnt/d/github-actions-locally/bin` into users .bashrc

Output:
```
Stage  Job ID                              Job name                             Workflow name             Workflow file                        Events
0      lua-unit-testing-on-ubuntu-latest   LuaUnit on Ubuntu-latest             Ubuntu Lua Unit Testing   ubuntu-latest-lua-unit-testing.yml   push,pull_request
0      lua-unit-testing-on-windows-latest  LuaUnit on Windows-latest (mingw32)  Windows Lua Unit Testing  windows-latest-lua-unit-testing.yml  push,pull_request
```

Thanks a lot for this public repo! 💯

**Comment by Ismoh** [comment]
I had to run it as root (not sure if this is meant to be), otherwise there were docker error: permission denied

But besided that it's working fine:
![image](https://user-images.githubusercontent.com/12631485/178733576-d1966986-d73d-4d5f-857d-2c3d6f1e8434.png)

**Comment by Lillebaek-BESTSELLER** [comment]
Just in case anybody ends up here, from wanting to run act or the act extension for vs code on WSL. The problem of having to run act as root user stems from docker permissions, and can be fixed like in this Stack overflow case: https://stackoverflow.com/questions/48957195/how-to-fix-docker-got-permission-denied-issue
